tests/test_dcos_e2e/test_node.py

```diff
@@ -158,6 +158,33 @@ def test_send_file(
         result = dcos_node.run(args=args)
         assert result.stdout.decode() == content
 
+    def test_send_directory(
+        self,
+        dcos_node: Node,
+        tmpdir: local,
+    ) -> None:
+        """
+        It is possible to send a directory to a cluster node as the default
+        user.
+        """
+        content = str(uuid.uuid4())
+        local_dir_name = 'local_dir'
+        remote_dir_name = 'remote_dir'
+        file_name = 'example_file.txt'
+        local_dir = tmpdir.mkdir(local_dir_name)
+        local_file = local_dir.join(file_name)
+        local_file.write(content)
+        random = uuid.uuid4().hex
+        master_base_dir = '/etc/{random}'.format(random=random)
+        master_destination_dir = Path(master_base_dir) / remote_dir_name
+        dcos_node.send_file(
+            local_path=Path(local_dir),
+            remote_path=master_destination_dir,
+        )
+        args = ['cat', str(master_destination_dir / file_name)]
```

Comment:
Ok so to clarify, this puts the contents of the local dir into the remote dir - right?

Let's go with whatever is easiest for now, and change as we see how this interface plays out.

Reply:
However, and I know we discussed this, this isn't how I'd expect it to work seeing it now.

I'd expect this to be like `cp -r`.